app/src/main/java/org/kiwix/kiwixmobile/zimManager/ZimManageViewModel.kt

```diff
@@ -248,8 +248,8 @@ class ZimManageViewModel @Inject constructor(
       .subscribe(
         {
           kiwixService.library
-            .retry(5)
-            .subscribe(library::onNext) {
+            ?.retry(5)
+            ?.subscribe(library::onNext) {
```

Comment:
If it's nullable then we should handle exceptions.

Reply:
we are fetching from List so nullable condition will not arrived.